grad_cam.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2017 Ruth Fong. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities to compute SaliencyMasks."""
+import numpy as np
+import tensorflow as tf
+
+from saliency import SaliencyMask
+from skimage.transform import resize
```

Comment:
Since the core library is trying to avoid dependencies, how about using tf.image.resize_bilinear(...) instead of skimage